src/components/LocationSelector/LocationSelectorDialog/__tests__/LocationSelectorDialog.spec.tsx

```diff
@@ -11,24 +13,18 @@ describe('LocationSelectorDialog component', () => {
     const selectedLocations = [
         {
             id: 'ajdo-219a-j19v-0491',
-            description: 'Amsterdam',
             center: {
                 lng: 4.894539799999961,
                 lat: 52.3666969,
             },
-            radius: 42,
-            sliderLabel: '42km',
-        },
+        } as LocationSelectorLocation,
```

Comment:
This object doesn't contain all of the information required for the type LocationSelectorLocation

Reply:
:+1: